<launch>
  <node name="stim300driver" pkg="driver_stim300" type="stim300_driver_node" output="screen" />
  <param name = "device_name" value="/dev/ttyUSB0" type="string" />
  <param name = "stanard_deviation_of_gyro" value="0.0004" type="double" />
  <param name = "stanard_deviation_of_acc" value="0.004" type="double" />
  <param name = "sample_rate" value="200" type="double" />
  <param name = "DAVIS_Reset_Delay" value="2.0" type="double" />
</launch>
